milatools/cli/utils.py

```diff
@@ -139,14 +139,15 @@ def shjoin(split_command):
 class SSHConfig:
     """Wrapper around sshconf with some extra niceties."""
 
-    def __init__(self, path):
+    def __init__(self, path: str | Path):
```

Comment:
this will constrain the python version maybe more than you want too

Reply:
There's a `from __future__ import annotations` at the top, so this only limits to Python 3.7+

Reply:
(https://peps.python.org/pep-0563/ for reference)

Reply:
cool, didn't know about that import